src/contracts/interfaces/IEmissionsController.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.5.0;
+
+/// @title IEmissionsControllerErrors
+/// @notice Errors for the IEmissionsController contract.
+interface IEmissionsControllerErrors {
+    // TODO: Define with implementation.
+}
+
+/// @title IEmissionsControllerTypes
+/// @notice Types for the IEmissionsController contract.
+interface IEmissionsControllerTypes {
+    /// @notice Distribution types as defined in the ELIP.
+    /// @dev Ref: "Distribution Submission types may include: createRewardsForAllEarners, createOperatorSetTotalStakeRewardsSubmission, createOperatorSetUniqueStakeRewardsSubmission, EigenDA Distribution, Manual Distribution."
+    enum DistributionType {
+        RewardsForAllEarners,
+        OperatorSetTotalStake,
+        OperatorSetUniqueStake,
+        EigenDA,
+        Manual
+    }
+
+    /// @notice A Distribution structure containing weight, type and strategies.
+    /// @dev Ref: "A Distribution consists of N fields: Weight, Distribution-type, Strategies and Multipliers."
+    struct Distribution {
+        uint256 weight;
+        DistributionType distributionType;
+        bytes strategiesAndMultipliers;
+    }
+
+    /// @notice Configuration for the EmissionsController.
+    /// @dev Ref: "The amount of EIGEN minted weekly (inflation rate) is set by governance..."
+    struct EmissionsConfiguration {
+        uint256 inflationRate;
```

Comment:
This will be a global fixed value not settable by the Incentives Council, only by Protocol Council ELIP. settable below.